packages/neos-ui-redux-store/src/CR/Nodes/index.js

```diff
@@ -41,13 +49,41 @@ const focus = createAction(FOCUS, (contextPath, fusionPath) => ({contextPath, fu
  */
 const unFocus = createAction(UNFOCUS);
 
+/**
+ * Start node removal workflow
```

Comment:
I get your point, but I disagree here. Imho we should not split this core workflow into multiple reducers.

From the pov of a caller (e.g. a "delete" button), it is much more natural in terms of workflow to dispatch a `CR.Nodes.commenceRemoval` action, than a `UI.DeleteModal.show` action, simply because the caller is interested in a node being removed, not a modal being shown.